exposed-kotlin-datetime/src/main/kotlin/org/jetbrains/exposed/sql/kotlin/datetime/KotlinDateFunctions.kt

```diff
@@ -10,14 +10,17 @@ import org.jetbrains.exposed.sql.*
 import org.jetbrains.exposed.sql.Function
 import org.jetbrains.exposed.sql.vendors.H2Dialect
 import org.jetbrains.exposed.sql.vendors.MysqlDialect
+import org.jetbrains.exposed.sql.vendors.PostgreSQLDialect
 import org.jetbrains.exposed.sql.vendors.SQLServerDialect
 import org.jetbrains.exposed.sql.vendors.currentDialect
 import org.jetbrains.exposed.sql.vendors.h2Mode
 import java.time.OffsetDateTime
 import kotlin.time.Duration
 
 internal class DateInternal(val expr: Expression<*>) : Function<LocalDate>(KotlinLocalDateColumnType.INSTANCE) {
```

Comment:
Not sure why these classes are internal in the `exposed-kotlin-datetime` module but not in the `exposed-java-time` and `exposed-jodatime` modules. If it's is not necessary for them to be internal, please let me know and I'll change that.

Reply:
@e5l Which annotation do you mean?

Reply:
Actually, I just realised that it _should_ be internal so that it is not invoked with expressions that are not temporal. The functions under it ensure that it can only be invoked with expressions of temporals.